terraform/.terraform/terraform.tfstate

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
given the setup of the backend in s3 you don't need to check in the terraform state since that needs to be on S3.